Core/src/com/serotonin/m2m2/db/dao/PointValueDaoSQL.java

```diff
@@ -104,6 +104,21 @@
     private final PointValues pv = PointValues.POINT_VALUES;
     private final DataPoints dp = DataPoints.DATA_POINTS;
 
+    public static final String SYNC_INSERTS_SPEED_COUNTER_ID = "com.serotonin.m2m2.db.dao.PointValueDaoSQL.SYNC_INSERTS_SPEED_COUNTER";
+    private static final ValueMonitor<Integer> SYNC_INSERTS_SPEED_COUNTER = Common.MONITORED_VALUES.<Integer>create(SYNC_INSERTS_SPEED_COUNTER_ID)
+                    .name(new TranslatableMessage("internal.monitor.SYNC_INSERTS_SPEED_COUNTER_ID"))
+                    .value(0)
+                    .build();
+
+    public static final String ASYNC_INSERTS_SPEED_COUNTER_ID = "com.serotonin.m2m2.db.dao.PointValueDaoSQL.ASYNC_INSERTS_SPEED_COUNTER";
+    private static final ValueMonitor<Integer> ASYNC_INSERTS_SPEED_COUNTER = Common.MONITORED_VALUES.<Integer>create(ASYNC_INSERTS_SPEED_COUNTER_ID)
+            .name(new TranslatableMessage("internal.monitor.ASYNC_INSERTS_SPEED_COUNTER_ID"))
+            .value(0)
+            .build();
+
+    private static final EventHistogram syncCallsCounter = new EventHistogram(1000, 2);
```

Comment:
With these histograms, I find it better to use a larger bucket size, so rewrite all these to use a bucket size of 5000 and divide by 5 when setting the metric's value to get saves/s.

Also all static fields should use upper case with underscores:
```
SYNC_CALLS_COUNTER
```

And add this comment to both sets of fields:
```
//TODO Mango 4.3 These are static since this class is not a singleton and must be shared across instances 
```